### DISCUSSION TITLE
Changing IP Address for Sensor Nodes in Security Onion

### DISCUSSION THREAD

**Discussion by SliuzasLukas** [Security-Onion-Solutions/securityonion#13716] [Unsupported Versions]
I'm in the process of preparing multiple sensor nodes for a Security Onion deployment. However, these nodes are currently in a network environment different from the one they'll be in during production. My question is: will simply changing the IP address in /etc/network/interfaces (or the relevant network configuration file) be enough to ensure the sensors will function correctly once moved to the production network?

Thanks in advance!

**Comment by dougburks** [comment]
Please see the IP Address section of the documentation:
https://docs.securityonion.net/en/2.4/ip.html

**Reply by SliuzasLukas** [reply]
This doesn't work on distribution mode as it says in the documentation. I need for search/sensor node

**Reply by dougburks** [reply]
Yes, that's exactly why I shared the link. We don't support changing IP addresses on distributed deployments like that.

**Reply by SliuzasLukas** [reply]
I have seen that documentation before wrote here, but I was hoping that somebody had this issue also.
